assets/src/edit-story/components/dropTargets/provider.js

```diff
@@ -82,11 +82,25 @@ function DropTargetsProvider({ children }) {
         return;
       }
 
+      const {
+        scale = 100,
```

Comment:
Can we get these from `elements/x/index.js` `defaultAttributes`?

Reply:
Done in 0994bb0 though indirectly.